### DISCUSSION TITLE
Android TV

### DISCUSSION THREAD

**Discussion by Marcus-Schubert** [Radiokot/photoprism-android-client#59] [Ideas]
Hey, I just tested your app on Android phone and it's been really nice so far.

Additionally it would be great to have the app also available on Android TV. I installed the current version directly on the nVidia Shield and while it loads successfully, navigation is nearly impossible.

Do you have any plans to work on an Android TV enabled app? I can offer my support for testing :)

**Comment by Radiokot** [comment]
Hey Marcus.

Thank you for checking out the gallery. I would definitely like the app to be accessible on all reasonably sized devices, I'll try to fix issues you've faced.

Please, describe what is wrong with the navigation in a new [Issue](https://github.com/Radiokot/photoprism-android-client/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=)

**Reply by Marcus-Schubert** [reply]
Hey @Radiokot , thank you for the quick reply.

I just created the following issue: https://github.com/Radiokot/photoprism-android-client/issues/60
